### PR TITLE
Add true current tile to tile indicator plugin

### PR DESCRIPTION
**Pull request by jesse1412**
Same concept as from [this close PR](https://github.com/runelite/runelite/pull/2764).

In action:

https://streamable.com/4uir6

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 1]
```diff
@@ -54,6 +54,27 @@ default boolean highlightDestinationTile()
...
+	}
+
+	@ConfigItem(
+			keyName = "highlightCurrentTile",
+			name = "Highlight current tile",
+			description = "Highlights tile player is on"
+	)
+	default boolean highlightCurrentTile()
+	{
+		return true;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: This should be off by default

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 2]
```diff
@@ -54,6 +54,27 @@ default boolean highlightDestinationTile()
...
 
+	@Alpha
+	@ConfigItem(
+			keyName = "highlightCurrentColor",
+			name = "Color of current tile highlighting",
+			description = "Configures the highlight color of current destination"
+	)
+	default Color highlightCurrentColor()
+	{
+		return Color.GRAY;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: This should probably not be same color as destination color by default

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 3]
```diff
@@ -54,6 +54,27 @@ default boolean highlightDestinationTile()
...
 
+	@Alpha
+	@ConfigItem(
+		keyName = "highlightCurrentColor",
+		name = "Color of current tile highlighting",
+		description = "Configures the highlight color of current tile position"
+	)
+	default Color highlightCurrentColor()
+	{
+		return Color.cyan;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: use Color.CYAN

**Comment by MagicfTail** [comment]
This would make a lot of content easier. A few obvious examples are Solo raids, ToB maze, and Vorkath.

**Comment by jesse1412** [comment]
> This would make a lot of content easier. A few obvious examples are Solo raids, ToB maze, and Vorkath.

I would agree if the tile indicator plugin didn't already exist. This is probably less useful over all (the tile indicator plugin can provide the same information as this plugin where it counts and additionally shows you if you've misclicked the wrong tile).

**Comment by MagicfTail** [comment]
Yeah, that is already bad enough imo. No need to make it even more obvious.

**Comment by deathbeam** [comment]
I disagree with it making Vorkath easier, but idk about rest of points you mentioned.

**Comment by melkypie** [comment]
This was added in runelite/plugin-hub#215

**Comment by deathbeam** [comment]
Imo this should be merged to master and the external plugin removed

**Comment by raiyni** [comment]
Does someone want to fix this up then?

**Pull request merged**